npm/react/examples/nextjs/package.json

```diff
@@ -6,20 +6,21 @@
     "build": "next build",
     "build:static": "next build && next out",
     "check-coverage": "check-coverage components/Search.jsx && check-coverage pages/index.js && check-coverage pages/router.js",
-    "cy:open": "node ../../../../scripts/cypress open",
+    "cy:open": "node ../../../../scripts/cypress open-ct",
     "dev": "next",
-    "only-cove red": "only-covered components/Search.jsx pages/index.js pages/router.js",
+    "only-covered": "only-covered components/Search.jsx pages/index.js pages/router.js",
```

Comment:
?????

Reply:
Just fixing script name

Reply:
I know, my question was how was this ever working ??